EnglishFilter/sections/general_url.txt

```diff
@@ -81,6 +81,7 @@ porn*.net/bees.js|
 !+ NOT_OPTIMIZED
 /getads?
 !
+/bannerpop/uploads/*$image,script
```

Comment:
This rule can apply to many website.
https://www.todaykorea.co.kr/
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://datanet.co.kr/
https://www.entermedia.co.kr/ ( https://www.entermedia.co.kr/news/articleView.html?idxno=27002 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
http://www.constimes.co.kr/
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )